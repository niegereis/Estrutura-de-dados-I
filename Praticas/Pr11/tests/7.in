7
7 10 2 7 4 2 1 7
Alice 1624323 74 122.9
Miguel 2122797 61 83.5
Sophia 2927990 68 44.8
Arthur 2817488 52 86.8
Helena 2226319 36 85.8
Bernardo 1324916 68 51.0
Valentina 1311979 25 102.7
