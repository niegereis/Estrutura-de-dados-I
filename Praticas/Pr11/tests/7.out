[2927990] Sophia - 68 (44.8)
[1624323] Alice - 74 (122.9)
[2817488] Arthur - 52 (86.8)
[2122797] Miguel - 61 (83.5)
[2226319] Helena - 36 (85.8)
[1324916] Bernardo - 68 (51.0)
[1311979] Valentina - 25 (102.7)
